app/src/main/java/com/toniebalonie/tjiang11/tcrunch/activities/TeacherTicketDetailActivity.java

```diff
@@ -294,7 +294,7 @@ public boolean onCreateOptionsMenu(Menu menu) {
     public boolean onOptionsItemSelected(MenuItem item) {
 
         int id = item.getItemId();
-
+//option to select all tickets for a class to export all
```

Comment:
- Use the same indentation as the other lines.
- Most style guides will say to leave a space between '//' and the beginning of your comment. Also, capitalization is nice!
- Probably clearer to say 'Add an option to export all tickets for a class.' Full sentences are nice too when possible!
- The typical workflow would be to instead just file a GitHub bug https://github.com/tjiang11/Tcrunch/issues. Personally for this project, since I did it alone I just used a separate doc to track my work. Just do whatever best fits your work style!

With that all being said, you don't have to do any of them because those really only matter if you're collaborating with more people. But at the same time, it's nice to have pretty code for yourself to look at that you wrote :) And if you ever share this project on your resume, people may poke around!